aComer/user/models.py

```diff
@@ -1,3 +1,44 @@
 from django.db import models
+from aComer.fonda import models as fmodels
 
 # Create your models here.
+class Client(models.Model):
+    first_name = models.CharField(max_length=50)
+    last_name = models.CharField(max_length=50)
+    phone = models.CharField(max_length=25)
+    email = models.EmailField(unique=True)
+
+    def __str__(self) -> str:
+        return f"{self.first_name},{self.last_name}"
+
+class ClientAddress(models.Model):
+    alias = models.CharField(max_length=50)
+    street = models.CharField(max_length=75)
+    suburb = models.CharField(max_length=100)
+    municipality = models.CharField (max_length=50)
```

Comment:
what is `municipality`?

Reply:
https://www.google.com/search?q=municipality&oq=municipality&aqs=chrome..69i57j0l5j46j0i457j0l2.3312j0j7&sourceid=chrome&ie=UTF-8